src/content/smart-bulletins/035.md

```diff
@@ -0,0 +1,68 @@
+
+### OMB Circular A-123, Appendix B Chapter 5.3.1 – Statistical Reporting –
+Requested information in the template attached below shall be submitted, at a minimum, on an annual fiscal year basis. Agency-provided information shall be furnished to CCCM no later than 90 calendar days after the end of the Fiscal Year (FY). The due date for agency/organization submission of FY2020 agency statistical reporting to CCCM is January 31, 2021. 
+
+The Statistical Reporting requirement applies to all business lines under the GSA SmartPay program however; the following guidance is for transit benefit issuing programs only. All other agencies should see Smart Bulletin No.030 for reporting guidance. Transit benefit programs must report the following items for each business line:
+
+- Count of accounts, as defined by GSA; 
+- Total number of accounts across the charge card program as compared to the prior reporting period;
+- Total charge card dollars spent; and 
+- Total refunds earned.
+
+Specific requirements for the purchase cards business line are as follows:
+
+Purchase cards used for transit benefits
+- Number of transit benefit accounts with limits over the statutory limit;
+- Ratio of the number of confirmed violations reported pursuant to P.L. 112-194 as compared to the number of valid transactions within the reporting period (GSA's Center for Charge Card Management will obtain confirmed violation information from OMB/OFFM).
+
+### OMB Circular A-123, Appendix B Chapter 5.3.2 – Narrative Reporting –
+For agencies/organizations subject to the CFO Act, the following information should be updated annually on a fiscal year basis and provided to CCCM no later than 90 calendar days after the end of each fiscal year: 
```

Comment:
The requested information in the template attached below shall be submitted, at a minimum, on an annual fiscal year basis. Agency-provided information shall be furnished to CCCM no later than 90 calendar days after the end of the Fiscal Year (FY). The due date for agency/organization submission of FY2020 agency statistical reporting to CCCM is January 31, 2021.
The Statistical Reporting requirement applies to all business lines under the GSA SmartPay program; however, the following guidance is for transit benefit issuing programs only. All other agencies should see Smart Bulletin No.030 for reporting guidance. Transit benefit programs must report the following items for each business line:
Count of accounts, as defined by GSA;
Total number of accounts across the charge card program as compared to the prior reporting period;
Total charge card dollars spent; and
Total refunds earned.
Specific requirements for the purchase cards business line are as follows:
Purchase cards used for transit benefits
Number of transit benefit accounts with limits over the statutory limit;
Ratio of the number of confirmed violations reported pursuant to P.L. 112-194 as compared to the number of valid transactions within the reporting period (GSA’s Center for Charge Card Management will obtain confirmed violation information from OMB/OFFM).